core/src/impl/Kokkos_Core.cpp

```diff
@@ -100,45 +99,105 @@ void declare_configuration_metadata(const std::string& category,
   metadata_map[category][key] = value;
 }
 
-ExecSpaceManager& ExecSpaceManager::get_instance() {
+void combine(Kokkos::InitializationSettings& out,
+             Kokkos::InitializationSettings const& in) {
```

Comment:
Why do you need this function?

Reply:
It replaces `void combine(InitArguments& out, InitArguments const& in)` and we use it in `initialize(InitializationSettings const&)`